### PR TITLE
RawQuery affectsTables and observesTables

### LINKED ISSUE #342: Add observesTables to the RawQuery
`RawQuery` can be used not only to affect db, but also for querying:

```java
storIOSQLite
  .get()
  .cursor()
  .withQuery(new RawQuery.Builder()
    .query("SELECT * FROM users JOIN tweets ON tweet.user_id = users.user_id WHERE tweet.timestamp > ?")
    .args(System.currentTimeMillis() - 86400)
    .affectedTables("users", "tweets") // this query does not affect db!
    .build())
  .prepare()
  .createObservable()
  .subscribeOn(Schedulers.io())
  .observeOn(AndroidSchedulers.mainThread())
  .subscribe(cursor -> adapter.setCursor(cursor)); // will be updated if users or tweets will be changed
```

I think `tables` instead of `affectedTables` should be good.

@nikitin-da what do you think?

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #342 

Now `RawQuery` looks good!

@nikitin-da PTAL

### CODE REVIEW

File: storio-common/src/main/java/com/pushtorefresh/storio/internal/Queries.java [hunk 1]
```diff
@@ -67,4 +82,6 @@ private Queries() {
                 ? null
                 : list.toArray(new String[list.size()]);
     }
+
+
```
Comment (nikitin-da) [inline storio-common/src/main/java/com/pushtorefresh/storio/internal/Queries.java]: Are that empty lines for better formating?

File: storio-common/src/test/java/com/pushtorefresh/storio/internal/QueriesTest.java [hunk 2]
```diff
@@ -77,8 +77,8 @@ public void nonEmptyListToUnmodifiable() {
 
     @SuppressWarnings("ConstantConditions")
     @Test
-    public void listToUnmodifiableIsReallyUnmodifiable() {
-        List<String> unmodifiableList = Queries.listToUnmodifiable(Arrays.asList("1", "2", "3"));
+    public void listToUnmodifiableNullableListIsReallyUnmodifiable() {
```
Comment (nikitin-da) [inline storio-common/src/test/java/com/pushtorefresh/storio/internal/QueriesTest.java]: =)

**Comment by nikitin-da** [comment]
Good!

File: storio-common/src/main/java/com/pushtorefresh/storio/internal/Queries.java (hunk 1 above)
Comment (artem-zinnatullin) [inline storio-common/src/main/java/com/pushtorefresh/storio/internal/Queries.java]: I don't know :D

**Pull request merged**